manim/mobject/graph.py

```diff
@@ -158,6 +158,26 @@ def construct(self):
                                        (4, 7): {"stroke_color": RED}})
                 self.add(g)
 
+    You can also lay out a partite graph on columns by specifying
+    a list of the vertices on each side and choosing the partite layout.
+
+    .. note::
+
+        All of the vertices which are not in one of the sides you provide
+        will be put into another column.
```

Comment:
I feel that we can be a bit more explicit in this note:

```suggestion
        All vertices in your graph which are not listed in any of the partitions
        are collected in their own partition and rendered in the rightmost column.
```